src/test/java/edu/hillel/lesson24/DBStorageTest.java

```diff
@@ -0,0 +1,36 @@
+package edu.hillel.lesson24;
+
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class DBStorageTest {
+
+    private final StorageFactory storageFactory = new StorageFactory();
+    private Storage storage;
+
+    @Before
+    public void setUp() {
+        storage = storageFactory.createStorage(StorageType.DB);
+        storage.clearData();
+    }
+
+    @Test
+    public void dataTest() {
```

Comment:
How can I build this test if I don't have db server locally? Or how Jenkins builds the application if db server doesn't exist?

Reply:
I used docker and docker-compose to start db. Docker-compose also attached to the project& Init sql also.
As I understand yesterday homework - it's about testing, not about full infrastucture. 
Yes - we could use testcontainers, spring-boot, liquibase and so-on.
But I think our excercise was about testing functionality.
Also - we could have texting enviroment with test-db. And as local tests as jenkins could run tests on it.
I don't understand what is incorrect.......

Reply:
Okey, why you didn't commit your docker-compose file? 
There are two ways to deal with it
First one - manual, you will commit your docker-compose file and add readme with description how to run tests correctly.
Second one - automatic, you will add a maven docker-compose plugin which will be automatically up and down docker-compose with db during a tests run

Reply:
<img width="317" alt="Screenshot 2023-10-31 at 16 29 38" src="https://github.com/MichaelTim75/Hillel/assets/15746517/ad25a584-0dd4-454b-a258-b2d78b15b767">

This it PR which I can see